rskj-core/src/test/java/co/rsk/peg/bitcoin/ErpRedeemScriptTestUtils.java

```diff
@@ -0,0 +1,60 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.Utils;
+import co.rsk.bitcoinj.script.ScriptOpCodes;
+
+import java.math.BigInteger;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ErpRedeemScriptTestUtils {
+    public static void assertStandardMultiSigRedeemScript(byte[] redeemScript, List<BtcECKey> emergencyPublicKeys, int opNumMErpIndex) {
+        int erpThreshold = emergencyPublicKeys.size() / 2 + 1;
+
+        // Next byte should equal M, from an M/N multisig
+        assertEquals(ScriptOpCodes.getOpCode(String.valueOf(erpThreshold)), redeemScript[opNumMErpIndex]);
+
+        int erpPubKeysIndex = opNumMErpIndex + 1;
+        for (BtcECKey btcErpEcKey : emergencyPublicKeys) {
+            byte actualErpKeyLength = redeemScript[erpPubKeysIndex++];
+
+            byte[] erpPubKey = btcErpEcKey.getPubKey();
+            assertEquals(erpPubKey.length, actualErpKeyLength);
+            for (byte characterErpPubKey : erpPubKey) {
+                assertEquals(characterErpPubKey, redeemScript[erpPubKeysIndex++]);
+            }
+        }
+
+        // Next byte should equal N, from an M/N multisig
+        int nERPIndex = erpPubKeysIndex;
+        int actualNErpFederation = redeemScript[nERPIndex];
+        int expectedNErpFederation = emergencyPublicKeys.size();
```

Comment:
```suggestion
        int expectedN = emergencyPublicKeys.size();
```
i think no need to say `federation`, wdyt?

Reply:
done!